data/json/monsters/insect_spider.json

```diff
@@ -1073,6 +1232,63 @@
     "zombify_into": "mon_meat_cocoon_large",
     "extend": { "flags": [ "DESTROYS", "PUSH_MON", "PUSH_VEH" ] }
   },
+  {
+    "id": "mon_moth_fly_small",
+    "type": "MONSTER",
+    "name": { "str": "oversized moth fly", "str_pl": "oversized moth flies" },
+    "description": "A mutant moth fly the size of a rabbit.",
+    "copy-from": "mon_moth_fly",
+    "proportional": { "speed": 1.1, "morale": 0.67, "melee_dice_sides": 0.67 },
+    "volume": "2500 ml",
+    "weight": "2500 g",
+    "hp": 7,
+    "upgrades": { "age_grow": 30, "into": "mon_moth_fly" },
+    "extend": { "flags": [ "NO_BREED" ] }
+  },
+  {
+    "id": "mon_moth_fly",
+    "type": "MONSTER",
+    "name": { "str": "giant moth fly", "str_pl": "giant moth flies" },
+    "description": "A moth fly the size of a small dog.  It appears to pay you no mind.",
+    "default_faction": "insect",
+    "bodytype": "flying insect",
+    "species": [ "INSECT_FLYING" ],
+    "volume": "15 L",
+    "weight": "15 kg",
+    "hp": 23,
+    "speed": 155,
+    "material": [ "iflesh" ],
+    "symbol": "y",
+    "color": "light_gray",
+    "aggression": -50,
+    "morale": 30,
+    "melee_skill": 3,
+    "melee_dice": 1,
+    "melee_dice_sides": 3,
+    "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
```

Comment:
Why do you need to define 0 cut?

Reply:
> Why do you need to define 0 cut?

I am pretty sure it actually defaults to 1 otherwise, but I normally throw that field away regardless. I missed it.

Reply:
That used to be a thing ages ago for `melee_cut`, not for the damage instance definitions.